Automattic/wp-calypso
# Fix/default currency

Related to https://github.com/Automattic/gold/issues/27
Goes along D110717-code


## Proposed Changes

* Retrieve default Stripe currency for the connected account
* Display the default currency when creating a new plan
* Minimum prices now depend on the  connected account default currency and will double if the selected currency is not the default oner to prevent issue discussed in p81Rsd-1hN-p2

## Testing Instructions

* Patch D110717-code on your sandbox
* Sandbox public-api.wordpress.com
* Use sandbox Store
* Launch this
* Disconnect the Stripe account and connect to a new one by selected some random currency (You can also modify the line from D110717-coden your sandbox to return a random currency)
-> When adding a new payment plan, the default seelcted currency should be the one from the stripe account
-> Empty the value. The minimum amount should the one referenced [in this list](https://github.com/Automattic/wp-calypso/blob/29527baf12c7c57bf4e0df4bf392f3071c587d9e/client/my-sites/earn/memberships/add-edit-plan-modal.jsx#L38). 
-> Change the currency to another one 
-> Empty the value. The minimum amount should be **the double of the one referenced [in this list](https://github.com/Automattic/wp-calypso/blob/29527baf12c7c57bf4e0df4bf392f3071c587d9e/client/my-sites/earn/memberships/add-edit-plan-modal.jsx#L38). ** 

## Pre-merge Checklist
